$OpenBSD: patch-Externals_soundtouch_SoundTouch_h,v 1.1 2020/03/08 02:15:51 thfr Exp $

Avoid picking up SoundTouch.h from audio/soundtouch if installed.
CMakeLists.txt: "Unable to use system soundtouch library: We require shorts, not
floats"

Index: Externals/soundtouch/SoundTouch.h
--- Externals/soundtouch/SoundTouch.h.orig
+++ Externals/soundtouch/SoundTouch.h
@@ -69,8 +69,8 @@
 //
 ////////////////////////////////////////////////////////////////////////////////
 
-#ifndef SoundTouch_H
-#define SoundTouch_H
+#ifndef ExtSoundTouch_H
+#define ExtSoundTouch_H
 
 #include "FIFOSamplePipe.h"
 #include "STTypes.h"
